client/src/modules/dashboard/shared/components/RoleChangeModal.tsx

```diff
@@ -43,7 +43,7 @@ export const RoleChangeModal: React.FC<{
     <Modal
       modalProps={modalProps}
       title={title}
-      buttons={<Button type="submit">Change</Button>}
+      buttons={<Button type="submit" colorScheme="green">Change</Button>}
```

Comment:
```suggestion
      formButtonText="Change"
```
This will use the default green button from `Modal`.